homeassistant/components/system_log/__init__.py

```diff
@@ -107,7 +107,10 @@ def _safe_get_message(record: logging.LogRecord) -> str:
     try:
         return record.getMessage()
     except Exception:  # pylint: disable=broad-except
-        return f"Bad logger message: {record.msg} ({record.args})"
+        try:
+            return f"Bad logger message: {record.msg} ({record.args})"
+        except Exception as ex:  # pylint: disable=broad-except
+            return f"Bad logger message: {ex}"
```

Comment:
Should this be this exception or the exception that was raised from `getMessage()` ?

Reply:
In this case they were the same but logging the original is better. Adjusted in [home-assistant/core@`68eb6e8` (#90652)](https://github.com/home-assistant/core/pull/90652/commits/68eb6e8f043bac93540811c5f0b957938d0d03d3)